### PR TITLE
Adjusted bottom bar size.

### PR DESCRIPTION
**Pull request by AlbertoCasasOrtiz**
I have adjusted the bottombar so it should always be visible now. Also fixed a problem making the scrollers to appear in both desktop and mobile.

### CODE REVIEW

File: src/components/pages/BottomBar.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ import { observer } from 'mobx-react'
 
 function BottomBar() {
     return (
-        <Container>
+        <Container style={{height: '55px' }}>
             <Stack direction="row" color="primary" justifyContent="center">
```
Comment (aymanhab) [inline src/components/pages/BottomBar.tsx:15]: @AlbertoCasasOrtiz Is there a way around specifying number of px in code, both here and in the expression on ModelViewPage? The current fix seems to work as of now but the view in dev tools shows the bar cropped on an iPad (though could be a chrome bug too). Could these numbers be moved to css? If not I'll merge this and see if it pops up later. on other devices. Thank you.

**Review by aymanhab — DISMISSED** [review]
Modulo one comment LGTM.

**Comment by aymanhab** [comment]
@AlbertoCasasOrtiz is this still WIP? Thanks

**Comment by AlbertoCasasOrtiz** [comment]
Yes, the bar is still hidden in my phone. So I am trying to figure out how to adjust it properly on every device without hard-coding values.

**Comment by AlbertoCasasOrtiz** [comment]
Looks like it is solved not. We have to consider that, in order to make the viewer adjust to the height of the viewport, I am substracting its size, which in desktop is 67px. However, I have found in the documentation that it actually adapts to the device, having different size in phones in portrait/landscape modes:

https://www.muicss.com/docs/v1/css-js/appbar

I am not sure on how to make the size of the canvas adapt to this. Should we create a new issue for this? @aymanhab

**Comment by aymanhab** [comment]
Thanks @AlbertoCasasOrtiz yes we can open an issue for that. With that said does this PR actually changes anything or we should just shelf it for now? Thank you

**Comment by AlbertoCasasOrtiz** [comment]
Yes, this solves a few visual problems in desktop and mobile related to viewport size inconsistency across devices.

**Review by aymanhab — APPROVED** [review]
Thanks @AlbertoCasasOrtiz Merging

**Pull request merged**